content/_data/changelogs/weekly.yml

```diff
@@ -1562,6 +1562,55 @@
     # pull: 3153 too minor: Grammar fixes
     # pull: 3151 internal: Jenkinsfile
     # pull: 3158 internal: disable flaky test
+- version: "2.93"
+  date: 2017-12-03
+  changes:
+    - type: rfe
+      pull: 3135
+      message: >
+        Use Java NIO to read and write Unix file permissions by default.
+        The previous behavior can be restored by setting the Java system property <code>hudson.Util.useNativeChmodAndMode</code> to <code>true</code>.
+      references:
+        - issue: 36088
+        - url: https://wiki.jenkins-ci.org/display/JENKINS/Features+controlled+by+system+properties
+          title: Jenkins features controlled by system properties
+    - type: rfe
```

Comment:
Maybe a bugfix though

Reply:
Can't be, because it doesn't have an associated issue :-)